apps/src/templates/instructions/teacherFeedback/Rubric.jsx

```diff
@@ -28,13 +27,16 @@ class TeacherFeedbackRubric extends Component {
       rubric,
       performance,
       isReadonly,
-      disabledMode,
       onRubricChange,
       viewAs
     } = this.props;
 
-    const showFeedbackInputAreas =
-      !isReadonly && (!!performance || viewAs === ViewType.Teacher);
+    const studentViewingPerfFeedback =
+      !!performance && viewAs === ViewType.Student;
+
+    const showFeedbackInputAreas = !isReadonly || studentViewingPerfFeedback;
+
+    const expandByDefault = isReadonly && !studentViewingPerfFeedback;
```

Comment:
This seems confusing.  Is this all to calculate whether we want to expand by default?  If so, could you describe when we want to expand by default and why?

Reply:
It is confusing, I was hoping to simplify it. There are several different cases to consider, whether viewing as a teacher or student, whether there is feedback already or if the teacher is giving feedback. I've updated the logic, let me know if it's a bit more clear now.